client/trino-cli/src/test/java/io/trino/cli/TestFormatUtils.java

```diff
@@ -275,6 +275,9 @@ public void testFormatDataRate()
         assertEquals(formatDataRate(DataSize.ofBytes(864000), Duration.valueOf("10d"), true), "1B/s");
         assertEquals(formatDataRate(DataSize.ofBytes(8640000), Duration.valueOf("10d"), false), "10B");
         assertEquals(formatDataRate(DataSize.ofBytes(8640000), Duration.valueOf("10d"), true), "10B/s");
+
+        assertEquals(formatDataRate(DataSize.ofBytes(1), Duration.valueOf("0s"), false), "0B");
```

Comment:
If we eg transferred 1 MB within "zero time" (which isn't impossible given clock granularity), why would we want to print 0?

Reply:
I don't know; I'm simply honoring the intent of the original code written by @electrum